### ISSUE TITLE
Synology improvements

### ISSUE THREAD

**Issue by kenkendk** [duplicati/duplicati#1766]
- [x] Add a "loading" screen initially
- [ ] Fix showing "User" and "Home" folders in source picker 
- [x] Fix first load, detect missing XSRF and request one
- [x] Set TMP_DIR to a subfolder of the install dir to avoid filling up /tmp
##

**Comment by kenkendk** [comment]
A major improvement to the Synology package would be to register a proxy with nginx instead of the slow CGI proxy currently used.

And example config could be like this, from the Docker package:
```
location ~ ^/api/v1/ws {
        proxy_set_header X-Server-IP $server_addr;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Real-HTTPS $https;
        proxy_set_header X-Server-Port $server_port;
        proxy_set_header X-Real-Port $remote_port;
        proxy_set_header Host $http_host;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_set_header X-Forwarded-Host $http_host;
        proxy_http_version 1.1;
        proxy_read_timeout 3600s;

        proxy_pass http://127.0.0.1:8200;
}
```

**Comment by kenkendk** [comment]
Fixed with 36cccd9776c7a554008a94c01e941c9bcb136185
Creating new issue for the "home" folders.
